cognite/client/data_classes/relationships.py

```diff
@@ -85,8 +85,8 @@ def _validate_resource_types(self) -> "Relationship":
 
     @staticmethod
     def _validate_resource_type(resource_type: Optional[str]) -> None:
-        _RESOURCE_TYPES = {"asset", "timeseries", "file", "event", "sequence"}
-        if resource_type is None or resource_type.lower() not in _RESOURCE_TYPES:
+        supported_res_types = {"asset", "timeseries", "file", "event", "sequence"}
```

Comment:
Since we're making a change here anyway, let's make this a class variable so it's only instantiated once.

Reply:
Agreed 👍